modules/nw-coredns-egress-firewall.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/configuring-egress-firewall-ovn.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-coredns-egress-firewall_{context}"]
+= Improved DNS resolution and resolving wildcard domain names
+
+There might be situations where the IP addresses associated with a DNS record change frequently or you might want to specify wildcard domain names in your egress firewall policy rules.
+
+:FeatureName: Improved DNS resolution for egress firewall rules
+include::snippets/technology-preview.adoc[]
+
+In this situation, the OVN-K cluster manager creates a `DNSNameResolver` custom resource object for each unique DNS name used in your egress firewall policy rules. This custom resource stores the following information:
+
+[source,yaml]
+.Example `DNSNameResolver` custom resource definition
+----
+apiVersion: networking.openshift.io/v1alpha1
+kind: DNSNameResolver
+spec:
+  name: www.example.com. <1>
+status:
+  resolvedNames:
+  - dnsName: www.example.com. <2>
+    resolvedAddress:
+    - ip: "1.2.3.4" <3>
+      ttlSeconds: 60 <4>
+      lastLookupTime: "2023-08-08T15:07:04Z" <5>
+----
+<1> the DNS name, this can either be a standard DNS name or a wildcard DNS name. For a wildcard DNS name, the DNS name resolution information contains all the DNS names that match the wildcard DNS name.
```

Comment:
```suggestion
<1> The DNS name. This can be either a standard DNS name or a wildcard DNS name. For a wildcard DNS name, the DNS name resolution information contains all of the DNS names that match the wildcard DNS name.
```